server/text_generation_server/layers/attention/rocm.py

```diff
@@ -173,13 +220,14 @@ def paged_attention(
 
     def attention(
         q,
-        k,
-        v,
-        cu_seqlens,
-        max_s,
+        key_cache: torch.Tensor,
+        value_cache: torch.Tensor,
+        seqlen: Seqlen,
+        block_tables: torch.Tensor,
         softmax_scale,
         window_size_left=-1,
         causal=True,
+        softcap=0.0,
```

Comment:
Type like above. (We are lacking types in a lot of places, but I usually try to add them for new arguments.)

Reply:
Added